packages/sdk/docs/README.md

```diff
@@ -22,6 +28,6 @@ yarn wasm:build[:node]:dev:multicore
 
 ## Usage
 
-See [examples](../examples)
+See [namada-sdkjs-examples](https://github.com/anoma/namada-sdkjs-examples)
```

Comment:
This repo appears to be a bit out of date compared to the local examples in `../examples`, which _must_ build when we generate docs, so they stay up-to-date. Maybe we should link to the these examples until the other repo is updated? I think it would be useful to have that other repo once our SDK package is published!

Reply:
Just a note - I've been ignoring changes to `docs/README.md` as it gets overwritten when generating docs, copying the other README here. I would maybe take the `docs/README.md` out for now until we can update it, and also see about changing the `typedoc` settings so it doesn't copy it over (haven't really looked at that yet)